resolve/resolve.go

```diff
@@ -688,7 +720,7 @@ func (r *resolver) expr(e syntax.Expr) {
 					r.errorf(pos, "multiple *args not allowed")
 				}
 				seenVarargs = true
-				r.expr(arg)
+				r.expr(unop.X)
```

Comment:
As I understand, `f(*x)` is interpreted as an expansion of arguments, not as a dereference. Makes sense, but the spec should be clear on how this ambiguity is resolved.